frontend/src/state/index.ts

```diff
@@ -183,3 +183,15 @@ export const getSchedulesFromState = (state: AppState): NamedSchedule[] =>
 export const getActiveScheduleFromState = (state: AppState): NamedSchedule => {
   return state.schedules.schedules[state.schedules.activeSchedule];
 };
+
+/**
+ * Get the active current class counter from the AppState
+ * @param state the AppState
+ */
+export const getCurrentClassCounterFromState = (state: AppState): number => {
+  if (!state.schedules.schedules[state.schedules.activeSchedule]) {
+    return state.schedule.present.currentClassCounter;
+  }
+  return state.schedules.schedules[state.schedules.activeSchedule].schedule
+    .present.currentClassCounter;
```

Comment:
I think you should just get this from state.schedule.present.currentClassCounter (?)
The schedule in schedule state will be a more up to date version of the active schedules (since the schedules in schedules state doesn't update until the user selects update schedule